tests/unit/amo/components/TestAddon.js

```diff
@@ -1185,39 +1190,39 @@ describe(__filename, () => {
     it('displays the developer name when add-on is an extension', () => {
       const root = renderMoreAddons({
         addon: fakeAddon,
-        addonsByAuthors: [{ ...fakeAddon, slug: 'another-slug' }],
+        addonsByAuthors: defaultAddonsByAuthors,
```

Comment:
I don't think it's helpful to use `defaultAddonsByAuthors` anywhere in here because it could introduce subtle bugs if the slugs end up conflicting.

Reply:
Yeah, I actually think this was a lazy stopgap to something better. I'll improve this.